packages/wagmi/exports/react.tsx

```diff
@@ -20,6 +22,50 @@ export function createWeb3Modal(options: Omit<Web3ModalOptions, '_sdkVersion'>)
   return modal
 }
 
+export function useWeb3ModalTheme() {
+  if (!modal) {
+    throw new Error('Please call "createWeb3Modal" before using "useWeb3ModalTheme" hook')
+  }
+
+  function setThemeMode(themeMode: ThemeMode) {
```

Comment:
See how `OpenOptions` are defined to inherit from function params directly, same approach should be applied to theme methods